proto/forwarding/forwarding_service.proto

```diff
@@ -110,12 +107,12 @@ service Forwarding {
 
   // NotifySubscribe subscribes to notifications.
   rpc NotifySubscribe(NotifySubscribeRequest) returns (stream EventDesc) {}
-}
 
-// PacketSink is a service that processes packets on the specified port.
-service PacketSink {
-  // PacketInject injects a packet into the specified port.
-  rpc PacketInject(PacketInjectRequest) returns (PacketInjectReply) {}
+  // PacketInject injects of packets into any port in the context.
+  rpc PacketInject(stream PacketInjectRequest) returns (PacketInjectResponse) {}
+
+  // PacketEject receives all packets sent to the CPU port.
+  rpc PacketEject(PacketEjectRequest) returns (stream PacketEjectResponse) {}
```

Comment:
Can you describe this a little more? Why is inject "any" port but eject only the "cpu" port? Maybe PacketSinkRequest is a better name? Just a suggestion

Reply:
renamed to PacketSinkSubscribe, so it matches close the NotifySubscibe. as they operate similar